landoapi/projects.py

```diff
@@ -81,38 +94,26 @@ def get_project_phid(project_slug, phabricator, allow_empty_result=True):
         none_when_empty=allow_empty_result,
     )
 
-    if project is None:
-        logger.warning(f"Could not find a project phid", extra=dict(slug=project_slug))
-        return None
-
-    return phabricator.expect(project, "phid")
+    value = phabricator.expect(project, "phid") if project else None
+    cache.set(key, value, timeout=DEFAULT_CACHE_KEY_TIMEOUT_SECONDS)
+    return value
 
 
-@cache.cached(key_prefix=SEC_PROJ_CACHE_KEY, timeout=DEFAULT_CACHE_KEY_TIMEOUT_SECONDS)
```

Comment:
IIRC you were going to split the caching changes into a separate PR.

Reply:
This was split into a different change.